txpipe/input_cats.py

```diff
@@ -228,12 +228,12 @@ def setup_photometry_output(self, photo_file, target_size):
 
 
 
-    def setup_metacal_output(self, metacal_file, target_size):
+    def setup_metadetect_output(self, metacal_file, target_size):
```

Comment:
mix use of metadetect and metacal? I'm guessing both are using the same file structure so it doesn't matter, but is there a way to make this less confusing?

Reply:
for the record - we chatted about this and the input_cats code now only generates metadetect-type files